feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-single-selection-base.component.ts

```diff
@@ -0,0 +1,144 @@
+import { ConfiguratorAttributeBaseComponent } from './configurator-attribute-base.component';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Configurator } from '../../../../core';
```

Comment:
Please import from actual file, not from index

Reply:
Done!